core/src/peschke/Convertible.scala

```diff
@@ -22,7 +22,10 @@ trait Convertible[In, Out] {
     convert(f(in))
 }
 
-trait ConvertibleLifts {
+trait ConvertibleNoOps {
+  implicit final def IdentityLift[A]: Convertible[A, A] = identity(_)
+}
+trait ConvertibleLifts extends ConvertibleNoOps {
```

Comment:
Personally I prefer a new line between type definitions (here and rest of the file). Any reason you prefer skipping new lines?

Reply:
I usually skip the blank line between traits that are stacked onto an object for the purpose of controlling implicit priority, because I consider them part of the object definition.